test/mock-connector.py

```diff
@@ -38,6 +39,27 @@
     level=logging.INFO,
     datefmt='%Y-%m-%d %H:%M:%S')
 
+def __parse_update_message(msg_file):
+    if msg_file:
+        with open(msg_file) as fd:
+            msg = json.load(fd)
+    else:
+        msg = {
+            "id": "0123456789abcdef",
+            "data": [{"sensor_id": 253, "data": True}]
+        }
+    return json.dumps(msg)
+
+def __parse_request_message(msg_file):
+    if msg_file:
+        with open(msg_file) as fd:
+            msg = json.load(fd)
+    else:
+        msg = {
```

Comment:
Create a global variable `defaultMsgRequest`

Reply:
Is not necessary in my opinion, if the function __parse_*_message exist.